07-custom-jobs.Rmd

```diff
@@ -0,0 +1,79 @@
+```{r, include = FALSE}
+ottrpal::set_knitr_image_path()
+```
+
+# Custom jobs
+
+```{r, echo=FALSE, fig.alt='We are on the fifth step of the pathway.', out.width = '80%', fig.align = 'center'}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1BQxrVYdKZTbpCaF-i_q9w7s9x034lEXpQZDU-Sl09cs/edit#slide=id.gff2211b72f_1_229")
+```
+
+Once you are comfortable submitting simple jobs, we point out ways customize your jobs to make best use of the computing infrastructure: perhaps you want to change the number of CPUs or RAM to request, or you want to monitor the status of a job running. This section highlights some common requests people use on the Fred Hutch Slurm system.
+
+## Requesting Specific Computing Resources
+
+By default, when you ran `sbatch 01.sh` in the previous chapter, `sbatch` requested for one CPU on a computing node for your task. Let's change the number of CPUs requested. You can change the number of requested CPUs based on the `-c` option: if you want to request 4 CPUs for your job, you would enter the following:
```

Comment:
Could we suggest `--cpus-per-task=<ncpus>` instead of `-c` since it's more descriptive? 

Could also create a callout box that points to the slurm options here : https://slurm.schedmd.com/sbatch.html. E.g., "There are lots of ways to customize your job submission in Slurm. You can even have it email you when your job is finished running! Check out the options [here]"

Reply:
I'll use both! 

I'm hesitant to link directly to SLURM's documentation as not all of them are available for Fred Hutch cluster.